plugins/services/src/js/service-configuration/PodGeneralConfigSection.js

```diff
@@ -61,7 +62,23 @@ function getInstances(appConfig) {
   return null;
 }
 
-module.exports = ({appConfig}) => {
+function getEditLink(tabViewID, handleEditClick) {
```

Comment:
I see a repeating pattern here. Perhaps introduce `<ConfigurationMapEditLink />` ? Or separate this into a utility function?